recipes/sentry-crashpad/all/conanfile.py

```diff
@@ -141,13 +143,20 @@ def package_info(self):
 
         self.cpp_info.components["snapshot"].libs = ["crashpad_snapshot"]
         self.cpp_info.components["snapshot"].requires = ["client", "compat", "util", "mini_chromium"]
+        if self.settings.os == "Windows":
+            self.cpp_info.components["snapshot"].system_libs.append("PowrProf")
```

Comment:
Use lower case libaries for mingw.
```suggestion
            self.cpp_info.components["snapshot"].system_libs.append("powrprof")
```

Reply:
Fixed.